pom.xml

```diff
@@ -143,14 +143,21 @@
     <dependency>
       <groupId>org.mockito</groupId>
       <artifactId>mockito-core</artifactId>
-      <version>2.18.3</version>
+      <version>3.8.0</version>
```

Comment:
Can you make this a property? `mockito.version` and then use it for both this and the next dependency as I think they're released together so we want to keep using the same version for both.